tools/actions/commit-queue.sh

```diff
@@ -76,10 +80,18 @@ for pr in "$@"; do
     git node land --abort --yes
   else
     rm output
+    head=$(gitHubCurl "$(prUrl "$pr")" GET | jq '{ ref: .head.ref, remote: .head.repo.ssh_url, can_modify: .maintainer_can_modify }')
```

Comment:
Could we split the value right here to avoid the ` "$(echo "$head" ..` below so that it will be a bit cleaner. Perhaps like:
```bash
head=$(gitHubCurl "$(prUrl "$pr")" GET)
head_ref="$(echo "$head" | jq -r '.head.ref')"
remote_url="$(echo "$head" | jq -r '.head.repo.ssh_url')"
can_modify="$(echo "$head" | jq -r '.maintainer_can_modify')"

```